jwst/regtest/test_nirspec_brightobj.py

```diff
@@ -49,3 +52,31 @@ def test_nirspec_brightobj_spec2(run_tso_spec2_pipeline, fitsdiff_default_kwargs
     # Compare the results
     diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
     assert diff.identical, diff.report()
+
+
+@pytest.mark.bigdata
+def test_flat_field_step_user_supplied_flat(jail, rtdata_module, fitsdiff_default_kwargs):
+    """Test providing a user-supplied flat field to the FlatFieldStep"""
+    rtdata = rtdata_module
+    data = rtdata.get_data('nirspec/tso/nrs2_wavecorr.fits')
+    user_supplied_flat = rtdata.get_data('nirspec/tso/nrs2_interpolatedflat.fits')
+
+    data_flat_fielded = FlatFieldStep.call(data, user_supplied_flat=user_supplied_flat)
+    rtdata.output = 'flat_fielded_step_user_supplied.fits'
+    data_flat_fielded.write(rtdata.output)
+
+    rtdata.get_truth('truth/test_nirspec_brightobj_spec2/flat_fielded_step_user_supplied.fits')
+    diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
+    assert diff.identical, diff.report()
+
+
+@pytest.mark.bigdata
+def test_ff_inv(jail, rtdata_module, fitsdiff_default_kwargs):
```

Comment:
Same with this one.  Use `rtdata` instead of `rtdata_module`.  And `jail` is not necessary, as `rtdata` already includes the function-scoped `_jail` fixture.